docker-compose.yaml

```diff
@@ -23,7 +23,7 @@ x-common-environment: &common-environment
 
 x-content-publishing-env: &content-publishing-env
   START_PROCESS: content-publishing-api
-  FILE_UPLOAD_MAX_SIZE_IN_BYTES: 2000000000
+  FILE_UPLOAD_MAX_SIZE_IN_BYTES: 500000000
```

Comment:
it seems like we can only upload files up to 500MB based on current setup and redis